config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   resources :donations, only: [:create, :update, :edit]
   resources :donors, only: [:show, :index, :update, :edit]
-  resources :events, only: [:index, :create]
+  resources :events, only: [:index, :create, :show]
```

Comment:
Let's pick an arbitrary order for our actions, and use that order for all our routes. 😀